.github/workflows/build.yml

```diff
@@ -0,0 +1,31 @@
+name: Build
```

Comment:
You should always add a license header to each file, which includes the SPDX statement.
See also: https://github.com/edgehog-device-manager/edgehog-device-runtime/blob/main/src/commands.rs